tests/test_utils.py

```diff
@@ -29,3 +37,16 @@ def test_l2_normalize():
 )
 def test_batch_sequences(seqs, expected):
     assert batch_sequences(seqs) == expected
+
+
+@given(seq=st.text(alphabet=proposal_valid_letters, min_size=1))
+def test_propose_string(seq):
+    new_seq = propose(seq)
+    assert new_seq != seq
+
```

Comment:
Should we maybe explicitly assert that the two seqs are of same length?

Reply:
Good catch. I updated the test.

Reply:
Done.